test/_init_tests.js

```diff
@@ -207,10 +207,31 @@ describe.configure = function() {
   return new TestConfig(describe);
 };
 
+const realIt = it;
+it = function(message, runnable) { // eslint-disable-line no-native-reassign, no-undef
+  if (!runnable || !runnable.constructor
+      || runnable.constructor.name !== 'GeneratorFunction') {
+    return realIt(message, runnable);
+  }
+  return realIt(message, done => {
+    const iterator = runnable();
+    const interval = setInterval(() => {
```

Comment:
`setTimeout` every time the function yields would be more appropriate. This can end up causing the test code to run again before it's really intended.